package.json

```diff
@@ -32,8 +32,8 @@
     "moment": "^2.24.0",
     "prop-types": "^15.7.2",
     "query-string": "^6.7.0",
-    "react": "^16.8.4",
-    "react-dom": "^16.8.4",
+    "react": "^16.9.0",
+    "react-dom": "^16.9.0",
```

Comment:
Upgrading `react` was necessary to properly test `MarkCompleteModal` which makes use of hooks.